src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1147,6 +1209,92 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
             {
                 scriptConfig.SwaggerEnabled = (bool)swaggerEnabled;
             }
+
+            ApplyLoggerConfig(config, scriptConfig);
+            ApplyApplicationInsightsConfig(config, scriptConfig);
+        }
+
+        internal static void ApplyLoggerConfig(JObject configJson, ScriptHostConfiguration scriptConfig)
+        {
+            scriptConfig.Filter = new LogCategoryFilter();
+            JObject configSection = (JObject)configJson["logger"];
+            JToken value;
+            if (configSection != null)
+            {
+                JObject filterSection = (JObject)configSection["categoryFilter"];
+                if (filterSection != null)
+                {
+                    if (filterSection.TryGetValue("defaultLevel", out value))
+                    {
+                        LogLevel level;
+                        if (Enum.TryParse(value.ToString(), out level))
+                        {
+                            scriptConfig.Filter.DefaultLevel = level;
+                        }
+                    }
+
+                    if (filterSection.TryGetValue("categoryLevels", out value))
+                    {
+                        scriptConfig.Filter.CategoryLevels.Clear();
+                        foreach (var prop in ((JObject)value).Properties())
+                        {
+                            LogLevel level;
+                            if (Enum.TryParse(prop.Value.ToString(), out level))
+                            {
+                                scriptConfig.Filter.CategoryLevels[prop.Name] = level;
+                            }
+                        }
+                    }
+                }
+
+                JObject aggregatorSection = (JObject)configSection["aggregator"];
+                if (aggregatorSection != null)
+                {
+                    if (aggregatorSection.TryGetValue("batchSize", out value))
+                    {
+                        scriptConfig.HostConfig.Aggregator.BatchSize = (int)value;
+                    }
+
+                    if (aggregatorSection.TryGetValue("flushTimeout", out value))
+                    {
+                        scriptConfig.HostConfig.Aggregator.FlushTimeout = TimeSpan.Parse(value.ToString());
+                    }
+                }
+            }
+        }
+
+        internal static void ApplyApplicationInsightsConfig(JObject configJson, ScriptHostConfiguration scriptConfig)
```

Comment:
You should have tests for this. For all our other config blocks, we have parsing unit tests (in ScriptHostTests I believe)